recipe/meta.yaml

```
@@ -48,19 +53,20 @@ requirements:
     - make                          # [unix]
     - m2-make                       # [win]
     - {{ compiler('c') }}           # [unix]
-    - {{ stdlib('c') }}             # [unix]
     - {{ compiler('m2w64_c') }}     # [win]
-    - {{ stdlib('m2w64_c') }}       # [win]
     - autoconf                      # [unix]
     - automake                      # [unix]
     - gettext                       # [unix]
     - libtool                       # [unix]
     - sed                           # [unix]
   host:
-    - xorg-xorgproto
     - libxcb
     - pthread-stubs
+    - xorg-xextproto
+    - xorg-kbproto
+    - xorg-inputproto
     - xorg-util-macros
+    - xorg-xorgproto
     - xorg-xtrans
```

```suggestion
    - xorg-xtrans
  run:
    - xorg-xorgproto
```
This is based on issues discovered in AnacondaRecipes/libxkbfile-feedstock#1. We should probably test a bit and see if these other message packages should also be included in the run environment.